### ISSUE TITLE
Bug: "Failed to assemble FASM Mnemonics"

### ISSUE THREAD

**Issue by datacompboy** [Reloaded-Project/Reloaded-II#198]
Trying to run No Man's Sky via Reloaded 2 (no mods!) I've get:

```
[18:15:12] [Reloaded] Create Loader | Time: 42ms
[18:15:12] [Reloaded] Loaded: reloaded.sharedlib.hooks in 6ms
[18:15:12] Warning: Steam Stub (Embedded Steam DRM) found.
This means EXE is encrypted at launch. Support for bypassing this DRM is experimental.
If you find issues, remove the DRM using `Steamless` or try using ASI Loader `Edit Application -> Deploy ASI Loader`
[18:15:12] [Reloaded] DRM Requiring Delayed Initialization (SteamStub) Found.
Reloaded will try to initialize late to bypass this DRM.
Please note this feature is experimental.
If you encounter issues, report and/or try ASI Loader `Edit Application -> Deploy ASI Loader`
[18:15:12] [Reloaded] Failed to Load Reloaded-II.
Failed to assemble FASM Mnemonics: Error name: ValueOutOfRange, Line Number: 29, Result: Error
   at Reloaded.Assembler.Assembler.Assemble(String, UInt16 )
   at Reloaded.Hooks.ReloadedHooks.CreateAsmHook(String[] asmCode, Int64 functionAddress)
   at Reloaded.Mod.Loader.Utilities.DelayInjector.CreateHook(Int64 address, Int32 dllOrdinal, Int32 functionOrdinal, IReloadedHooks hooks)
   at Reloaded.Mod.Loader.Utilities.DelayInjector..ctor(IReloadedHooks hooks, Action action, Logger logger)
   at Reloaded.Mod.Loader.EntryPoint.LoadMods(IReloadedHooks hooks)
   at Reloaded.Mod.Loader.EntryPoint.<>c__DisplayClass14_0.<SetupLoader2>b__1()
   at Reloaded.Mod.Loader.EntryPoint.ExecuteTimed(String text, Action action)
   at Reloaded.Mod.Loader.EntryPoint.SetupLoader2(EntryPointParameters* parameters)
   at Reloaded.Mod.Loader.EntryPoint.SetupLoader(EntryPointParameters* parameters)
A log is available at: C:\Users\datac\AppData\Roaming\Reloaded-Mod-Loader-II\Logs\2023-03-11 17.15.12 ~ NMS.txt
[18:15:12] [R-II SteamHook] SteamAPI_RestartAppIfNecessary hooked successfully.
[18:15:12] [R-II SteamHook] SteamAPI_IsSteamRunning hooked successfully.
[18:15:12] [R-II SteamHook] Found Steam Library Entry with Id 275850. Dropping steam_appid.txt.
[18:15:12] [Reloaded] Setting Up Hooks (Async) | Time: 167ms
```

Same effect with ASI loader injected.

Tried with Obduction -- and it working fine...

**Comment by Sewer56** [comment]
I'll look at it soon-ish, gimme a while.
Are you using latest Steam ver?

**Comment by Sewer56** [comment]
I downloaded the game; and I can reproduce this; gimme a while to investigate

**Comment by Sewer56** [comment]
This should be fixed in 1.23.10; which should be available in around 10 minutes. If after update it isn't fixed; reopen this issue.
